providers/lmgtfy/lmgtfy.go

```diff
@@ -0,0 +1,20 @@
+package google
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("lmgtfy", &LMGTFYProvider{})
+}
+
+// LMGTFYProvider implements the Provider interface.
+type LMGTFYProvider struct{}
```

Comment:
Indeed. Using packages for what they're supposed to be used for...

On that note, there's really no need to export Provider, and then the comment can be deleted too.